Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.Amigo;
```

Comment:
The package name should change to "Amigo", not "AMIGO".

Reply:
AMIGO is an abbreviation, similar to PID. So I changed all the "Amigo" into "AMIGO". Please let me know if it makes sense.